terraform/variables.tf

```diff
@@ -71,3 +71,25 @@ locals {
 
   azure_credentials = try(jsondecode(var.azure_credentials), null)
 }
+
+# Azure Resource Variables
+
+# variable azure_environment_name {}
```

Comment:
Could we use paas_app_environment instead? it's the same value
See https://github.com/DFE-Digital/register-trainee-teachers/blob/main/terraform/variables.tf#L3

Reply:
I did think about this but was thinking to keep the Azure stuff distinct for now. But I have updated it as per your comment

Reply:
I see the point about paas. But we can think of it as "Use the same environment name is azure as the paas one".
When we migrate off paas, we can rename this variable.